src/server/src/main/java/io/cassandrareaper/service/MetricsService.java

```diff
@@ -93,4 +107,43 @@ public static MetricsService create(AppContext context) {
 
     return metrics;
   }
+
```

Comment:
can the exception list in the following three methods be cleaned up?
i'm thinking it should be only `ReaperException`, and other exceptions resolved as either `IllegalArgumentException` or `IllegalStateException` ?

Reply:
Indeed, only `ReaperException` was necessary after all. I've cleaned up the `InterruptedException` from the signature.

Reply:
was that change pushed?